admin/services/data-access/pupil-access-arrangements.data.service.js

```diff
@@ -15,8 +15,12 @@ const pupilAccessArrangementsDataService = {}
 pupilAccessArrangementsDataService.sqlFindPupilAccessArrangementsByPupilId = async function (pupilId) {
   const sql = `
   SELECT
-  *
-  FROM ${sqlService.adminSchema}.[pupilAccessArrangements]
+  *, pfs.code AS pupilFontSizeCode, pcc.code AS pupilColourContrastCode
+  FROM ${sqlService.adminSchema}.[pupilAccessArrangements] paa
+  LEFT JOIN ${sqlService.adminSchema}.[pupilFontSizes] pfs
```

Comment:
Can we stick to `INNER` and `OUTER` syntax please

Reply:
@GuyHarwood  updated to `FULL OUTER`